pkgs/development/tools/build-managers/wafHook/setup-hook.sh

```diff
@@ -10,6 +10,11 @@ wafConfigurePhase() {
         wafConfigureFlags="${prefixKey:---prefix=}$prefix $wafConfigureFlags"
     fi
 
+    if ! command -v python >/dev/null 2>&1; then
+	echo "Adding python to path"
```

Comment:
NB: it would be cool to have a nix bash library for this kind of situation. I had added nix_debug / nix_info in pkgs/development/interpreters/lua-5/setup-hook.sh for instance (that act conditionnaly on NIX_DEBUG). 

the indentation seems off.

Reply:
```suggestion
	        echo "Adding python to path"
```